data/successors/successor news.txt

```diff
@@ -0,0 +1,277 @@
+			"An advertisement plays showing a variety of attractive and expensive-looking jewellery, made of gold, silver, and other stranger metals which you cannot recognize. A variety of models display it wrapped around their arms or other parts of their body, gently undulating as the camera moves."
+			"A display shows an advertisement for one of the Successor universities, showing elegant, sculptural buildings emerging from the ground under a rain-soaked sky."
+			"An field of large, floral plants are shown growing long strings of gleaming metal which are quickly harvested by agricultural robots."
+			"A series of images displaying several models flash one after another so quickly that it is hard to tell what is actually being advertised."
+			"A short video clip is shown advertising a performance in a local concert hall. The stage is divided into two sections: one is underwater and the other is dry."
+			"Some sort of interactive game that you can't make sense of captivates the attention of a group of young Successors."
+			"An eye covering vaguely resembling a contact lens is shown being applied to the eyes of several models, changing their color and shape when applied."
+			"A shipbuilding organization advertises a members' discount on Kvelqde shuttles. They make sure to note that their operation is supported by one of the High Houses."
+			"An independent pilot briefly advertises their courier services in a pre-recorded message, offering guaranteed fastest-possible-delivery on packages under ten tons."
+
+
+news "local news quiet"
+	to show
+		has "language: Successor"
+	location
+		attributes successor
+		attributes quiet
+		not attributes station uninhabited
+	name
+		word
+			"Local News Report"
+	message
+		word
+			"A broadcast quietly lists out the details of a recent weather control unit failure, which resulted in a brief evacuation but no injuries. A photo shows a few dozen Successors gathered together along a shoreline while several ships swarm in the distance."
+			"A muted advertisement plays, showing architectural renders of several small underwater homes with walls that change color and texture from simulated wood, to simulated shell, to even a dark, amorphous metal resembling the hull of a starship."
+			"A short broadcast discusses a recent thunderstorm, showing several colorful pictures of lightning and rain taken by locals during the event."
+			"A text-only broadcast discusses a recent touch-based art performance. The script shimmers as you move your head around, the translation device doing its best to transcribe it."
+			"A broadcast shows a brief interview with a local florist, who shows off a collection of their living-metal plants."
+			"A formal document is shown with a column of text to the side of it. This describes it as a new trade ordinance, and lists its effects on independent merchants."
+			"A brief program on erosion control and restoration plays for a duration."
```

Comment:
You've already told me it's brief, I don't also need to know that it played for some ambiguous amount of time.
```suggestion
			"A brief program on erosion control and restoration plays."
```
You could also add another sentence, of course, if you felt like it.